perllib/FixMyStreet/Cobrand/Thamesmead.pm

```diff
@@ -66,11 +66,16 @@ sub updates_restriction { FixMyStreet::Cobrand::UKCouncils::updates_restriction(
 
 sub base_url { FixMyStreet::Cobrand::UKCouncils::base_url($_[0]) }
 
+sub contact_email {
+    my $self = shift;
+    return $self->feature('contact_email');
```

Comment:
Could this use `FixMyStreet::Cobrand::UKCouncils:: contact_email`, similar to the methods above?

Reply:
$self->next::method(); Not working in a quirk I don't understand, so leaving this for now.